src/graphics/backend_gfx/mod.rs

```diff
@@ -128,6 +128,8 @@ impl Gpu {
         font: &mut Font,
         target: &TargetView,
         depth: &DepthView,
+        _target_width: u32,
+        _target_height: u32,
```

Comment:
I think `glyph_brush` can be improved to be target size agnostic. I will try to make a PR with the idea, but there will be breaking changes.

Reply:
Related issue https://github.com/alexheretic/glyph-brush/issues/63